pmm/pmm2-release.groovy

```diff
@@ -9,9 +9,10 @@ pipeline {
     }
 
     environment {
-        CLIENT_IMAGE     = "perconalab/pmm-client:${VERSION}-rc"
-        SERVER_IMAGE     = "perconalab/pmm-server:${VERSION}-rc"
-        PATH_TO_CLIENT   = "testing/pmm2-client-autobuilds/pmm2/${VERSION}/pmm-${VERSION}/${PATH_TO_CLIENT}"
+        CLIENT_IMAGE            = "perconalab/pmm-client:${VERSION}-rc"
+        SERVER_IMAGE            = "perconalab/pmm-server:${VERSION}-rc"
+        PATH_TO_CLIENT_AMD64    = "testing/pmm2-client-autobuilds-amd/pmm2/${VERSION}/pmm-${VERSION}/${PATH_TO_CLIENT_AMD64}"
```

Comment:
I believe we can get rid of `.../pmm2/...` subpath, since the full path is already sufficiently descriptive in terms of what we store there.

Reply:
OK, let's consider shortening this in the `authobuild` pipeline.